lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_generate_po_file_from_metadata.rb

```diff
@@ -0,0 +1,100 @@
+require 'fastlane/action'
+require 'gettext/po'
+require 'set'
+
+module Fastlane
+  module Actions
+    class AnGeneratePoFileFromMetadataAction < Action
```

Comment:
Nitpick: the `An` prefix for our Android actions is kind of legacy. Nowadays we name our actions that are Android-specific with the `Android` prefix, not just `An`. (we never got to rename the existing legacy actions with that new convention, but as soon as we re-implement them we try to adopt the new convention)